src/legacy/List/List.component.js

```diff
@@ -268,6 +268,10 @@ export class ListHybrid extends React.Component {
         this.setState({ visibleColumns });
     };
 
+    _updateSearch = query => {
```

Comment:
A very minor comment, I might call `_updateSearch` something like `_updateQuery`, since `query` is the variable updated, and it would be more consistent with the naming of other similar functions in that file. But I'm not opposed to its current name.

Reply:
Yes, it makes sense. Done.